mobile/src/common/types/search/search-input.type.ts

```diff
@@ -0,0 +1,5 @@
+type SearchInput = {
```

Comment:
Let's move it to components/common/search/common/types/search-payload.type.ts
```suggestion
type SearchPayload = {
```